apps/docs/docs/troubleshoot.md

```diff
@@ -0,0 +1,20 @@
+# Troubleshoot 
```

Comment:
I love this!

Reply:
yess! We can add to it with FAQs, I'll add the react query error when you haven't put a QueryClientProvider :)